### PR TITLE
Update AssetId

### PR DESCRIPTION
**Pull request by AsceticBear**
0 is incorrect here. The KTON asset id is reserved as 1026.

### CODE REVIEW

File: runtime/darwinia/src/pallets/assets.rs [hunk 1]
```diff
@@ -19,6 +19,11 @@
 // darwinia
 use crate::*;
 
+/// List of the assets existed in this runtime.
+pub enum AssetIds {
+	KTON = 1026,
```
Comment (AurevoirXavier) [inline runtime/darwinia/src/pallets/assets.rs:24]: Kton

File: runtime/darwinia/src/pallets/assets.rs [hunk 2]
```diff
@@ -19,6 +19,11 @@
 // darwinia
 use crate::*;
 
+/// List of the assets existed in this runtime.
+pub enum AssetIds {
+	Kton = 1026,
```
Comment (AurevoirXavier) [inline runtime/darwinia/src/pallets/assets.rs:24]: Is this the same for all network?

Comment (AsceticBear) [reply runtime/darwinia/src/pallets/assets.rs:24]: yes, it is.

Comment (AurevoirXavier) [reply runtime/darwinia/src/pallets/assets.rs:24]: Might move this to runtime common later.

**Pull request merged**